databricks-sdk-java/src/main/java/com/databricks/sdk/service/provisioning/CustomTags.java

```diff
@@ -1,10 +0,0 @@
-// Code generated from OpenAPI specs by Databricks SDK Generator. DO NOT EDIT.
-
-package com.databricks.sdk.service.provisioning;
-
-/**
- * The custom tags key-value pairing that is attached to this workspace. The key-value pair is a
- * string of utf-8 characters. The value can be an empty string, with maximum length of 255
- * characters. The key can be of maximum length of 127 characters, and cannot be empty.
- */
-// type CustomTags Map<String,String>
```

Comment:
Since these were never used, this change shouldn't be breaking right?

Reply:
Correct.